inc/packages/admin/info.php

```diff
@@ -434,7 +436,7 @@ function get_action_button( MetadataDocument $doc, ReleaseDocument $release ) {
 	}
 
 	// Do we actually meet the requirements?
-	// $compatible = check_requirements( $release );
+	$compatible = check_requirements( $release );
 	$compatible = true;
```

Comment:
Should we be removing the `$compatible = true;`?

Reply:
You are correct.

Reply:
Cool. Wasn't sure if we were forcing `true` for testing other parts at this stage.